### PR TITLE
Add eulxml recipe

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Add eulxml recipe

https://conda-forge.org/docs/recipe.html

### CODE REVIEW

File: recipes/eulxml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - ply >=3.8
+    - lxml >=3.4
+    - six >=1.10
+  run:
+    - python
+    - setuptools
+    - ply >=3.8
+    - lxml >=3.4
+    - six >=1.10
+    - django ==1.8.19
```
[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/eulxml/meta.yaml]: This is a requirement, I just saw this library uses a deprecated method of Django which is available until ```1.8.19``` after this version they removed the method.

[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/eulxml/meta.yaml]: However, I will double check to see if just their test is using that or not.

File: recipes/eulxml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  name: 'eulxml'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ hash_value }}'
+
+build:
+  number: 0
```
[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/eulxml/meta.yaml]: I will try, in theory I think it could be

**Comment by marcelotrevisani** [comment]
@tadeu  or @nicoddemus , Could you help me here?

File: recipes/eulxml/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/eulxml/meta.yaml]: I was thinking... I need some selectors, so I can't use that

[the remark this answers is not in the document]
Comment (synapticarbors) [reply recipes/eulxml/meta.yaml]: If the selectors are just for package versions, that should be ok. It's only an issue if the selectors define different requirements for different platforms.

[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/eulxml/meta.yaml]: Ok! I will try with that

**Comment by marcelotrevisani** [comment]
@conda-forge-admin, please rerender

**Comment by marcelotrevisani** [comment]
@conda-forge-admin, please add noarch: python

**Comment by marcelotrevisani** [comment]
@synapticarbors , Could you please review that for me?

**Comment by marcelotrevisani** [comment]
Thanks @synapticarbors  !

**Pull request merged**